examples/Accept/Readme.md

```diff
@@ -59,37 +64,35 @@ Because of HTML5 File API differences across different browsers during the drag,
 
 Also, at this moment it's not possible to read file names (and thus, file extensions) during the drag operation. For that reason, if you want to react on different file types _during_ the drag operation, _you have to use_ mime types and not extensions! For example, the following example won't work even in Chrome:
 
-```
-<Dropzone
-  accept=".jpeg,.png"
->
-  {({ isDragActive, isDragReject }) => {
-    if (isDragActive) {
-      return "All files will be accepted";
-    }
-    if (isDragReject) {
-      return "Some files will be rejected";
-    }
-    return "Dropping some files here...";
-  }}
+```jsx harmony
+<Dropzone accept=".jpeg,.png">
+  {({ getRootProps, getInputProps, isDragActive, isDragAccept, isDragReject }) => (
+    <div {...getRootProps()} className="dropzone">
+      <input {...getInputProps()} />
+      <div>
+        {isDragAccept && "All files will be accepted"}
```

Comment:
Isn't this one going to render `false` if `isDragAccept` is false?

Reply:
Nope, it seems to not render `false`.